### PR TITLE
chore(base-cluster): update/traefik

### PR DESCRIPTION
**Pull request by tasches**

### CODE REVIEW

**Comment by tasches** [comment]
adaptions necessary in horizontal pod autoscaler

File: charts/base-cluster/templates/ingress/traefik.yaml [hunk 1]
```diff
@@ -115,8 +115,6 @@ spec:
         preferServerCipherSuites: true
         minVersion: VersionTLS12
 
-    podSecurityPolicy:
-      enabled: true
```
Comment (tasches) [inline charts/base-cluster/templates/ingress/traefik.yaml:119]: we can template it for older k8s version if required

**Review by mstroecker — APPROVED** [review]
LGTM

File: charts/base-cluster/templates/ingress/traefik.yaml (hunk 1 above)
Comment (mstroecker) [reply charts/base-cluster/templates/ingress/traefik.yaml:119]: If someone is at that k8s version, an update is overdue, right?

**Pull request merged**